packages/keyring-controller/CHANGELOG.md

```diff
@@ -6,6 +6,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [13.0.0]
+### Changed
+- Bump @metamask/eth-keyring-controller from 17.0.0 to 17.0.1 ([#3805](https://github.com/MetaMask/core/pull/3805))
+- **Breaking** updates `@metamask/keyring-api` to ^3.0.0 ([#3747](https://github.com/MetaMask/core/pull/3747))
+- feat: add methods to support ERC-4337 accounts ([#3602](https://github.com/MetaMask/core/pull/3602))
+- refactor: Update `@metamask/utils` and use `createDeferredPromise` from utils ([#3769](https://github.com/MetaMask/core/pull/3769))
```

Comment:
The use of `createDeferredPromise` is an internal change that doesn't need to be listed here. Note that in the PR itself, there is a "Changes" section that lists change entries for this PR.The suggested entry here is

```suggestion
- Bump `@metamask/utils` to `^8.3.0` ([#3769](https://github.com/MetaMask/core/pull/3769))
```